kong/db/schema/metaschema.lua

```diff
@@ -2,18 +2,21 @@
 -- @module kong.db.schema.metaschema
 
 local Schema = require "kong.db.schema"
+local constants = require "kong.constants"
 
 
 local setmetatable = setmetatable
 local assert = assert
 local insert = table.insert
 local pairs = pairs
+local ipairs = ipairs
 local find = string.find
 local type = type
 local next = next
 local keys = require("pl.tablex").keys
 local sub = string.sub
 local fmt = string.format
+local bundled_plugins = constants.BUNDLED_PLUGINS
```

Comment:
Suggest keeping the capitalization (`BUNDLED_PLUGINS`) when locallizing to make it easier to understand that a global variable is accessed.